tox.ini

```diff
@@ -64,7 +60,7 @@ commands =
     pytest {posargs:tests} \
         --cov-fail-under=85 \
         --cov-report=html \
-        --cov-report=xml \
+        --cov-report=xml
```

Comment:
are those \ needed then?

Reply:
Strange, I don't understand how that line change ended up there :thinking: But yes, that `\` should be necessary

Reply:
For the record, this happens without the `\`:
```
test run-test: commands[1] | --cov=expertsystem
ERROR: InvocationError for command could not find executable cov=expertsystem
```